queue_job/data/queue_data.xml

```diff
@@ -8,7 +8,9 @@
             <field name="numbercall">-1</field>
             <field ref="model_queue_job" name="model_id" />
             <field name="state">code</field>
-            <field name="code">model.requeue_stuck_jobs()</field>
+            <field
+                name="code"
+            >model.requeue_stuck_jobs(enqueued_delta=5, started_delta=-1)</field>
```

Comment:
While we are at it, 5 minutes for enqueued_delta is a lot. In practice (if #workers <= root channel capacity), a job will never be enqueued for more than 1 second.

Reply:
I've not changed anything about enqueued_delta. 

What are you thinking about? Maybe 1 minute should be enough?

Reply:
@Shide I think you refer to Pending jobs. Jobs are in an Enqueued state for a normally very short duration between the decision by the job runner to launch the job and the time the http worker has started it.

Reply:
And the `requeue` fonction actually sets the state to Pending, so yeah it's a bit confusing :shrug:

Reply:
Ok, I'll set it to 1 minute.